### PR TITLE
fixing failed resources in buildkite

### PR DESCRIPTION
**Pull request by soumyo13**
### Description

The buildkite was failing as 2 unit tests were failing. We fixed it.

### Issues Resolved

List any existing issues this PR resolves, or any Discourse or StackOverflow discussion that's relevant

### Check List
Please fill box or appropriate ([x]) or mark N/A.
- [ ] New functionality includes integration tests/controls
- [ ] New Terraform resources
- [ ] Documentation provided or updated for resources 
- [ ] All Integration Tests pass
- [x] All Unit Tests pass
- [x] `rake lint` passes
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco>

### CODE REVIEW

File: libraries/aws_mq_configuration.rb [hunk 1]
```diff
@@ -4,37 +4,37 @@
...
+  desc 'Returns information about the specified configuration.'
+  example <<-EXAMPLE
+    describe aws_mq_configuration(configuration_id: 'CONFIGURATION_ID') do
       it { should exist }
     end
-  "
+  EXAMPLE
 
   def initialize(opts = {})
-    opts = { configuration_id: opts } if opts.is_a?(String)
```
Comment (sathish-progress) [inline libraries/aws_mq_configuration.rb:16]: why is this removed?

File: libraries/aws_mq_configuration.rb [hunk 2]
```diff
@@ -4,37 +4,37 @@
...
-    @configurations = resp.to_h
-    create_resource_methods(@configurations)
-  end
-
-  def configuration_id
-    return nil unless exists?
-    @configurations[:configuration_id]
+    catch_aws_errors do
+      resp = @aws.mq_client.describe_configuration({ configuration_id: opts[:configuration_id] })
+      @res = resp.to_h
```
Comment (sathish-progress) [inline libraries/aws_mq_configuration.rb:21]: `@configurations` is a better variable name than `@res`

File: libraries/aws_mq_configuration.rb [hunk 3]
```diff
@@ -4,37 +4,37 @@
...
+      @arn = @res[:arn]
+      create_resource_methods(@res)
+    end
   end
 
   def resource_id
-    @configurations? @configurations[:configuration_id] : @display_name
+    @res? @arn: @display_name
```
Comment (sathish-progress) [inline libraries/aws_mq_configuration.rb:28]:
```suggestion
    return unless exists?
    
    @arn || @display_name
```

**Review by sathish-progress — CHANGES_REQUESTED** [review]

File: libraries/aws_mq_configuration.rb (hunk 3 above)
Comment (soumyo13) [reply libraries/aws_mq_configuration.rb:28]: Changes done

File: libraries/aws_mq_configuration.rb (hunk 2 above)
Comment (soumyo13) [reply libraries/aws_mq_configuration.rb:21]: Changes done
